README.md

```diff
@@ -28,7 +28,7 @@ jobs:
           bufVersion: v0.31.1
           protocPlugins:
             go@v1.25.0 go-grpc@v1.0.1 python_betterproto@2.0.0b2 ts-proto@1.39.0
-            grpc-java@1.34.0
+            grpc-java@1.34.0 envoy-validate@v0.6.1
```

Comment:
I am not :100: convinced by the name. The plugin itself is not related to envoy itself, so it might be misleading (it was for me).
But only `validate` might be to generic? Still better than `envoy-validate` IMO.

Reply:
Yeah totally agree with you on this I tried to match the name of the repo as that seemed to be the pattern going with others but they are obviously more appropriately named. 

I could perhaps call it something like `proto-validate` (this could perhaps be too generic again) or `envoy-proto-validate` then add a little info section about the plugin why it's named the way it is and link back to the actual plugin repo?

Reply:
Sure, the problem is their naming in the fist place :smile: especially when you consider `buf validate`
What about `gen-validate`?

Reply:
I think maybe go with `envoy-protoc-gen-validate` it's a bit wordy but I'm sure the majority of people using this would just end up copy/pasting the code anyways so I don't think it would cause many issues being that long. What're your thoughts on this name?

Reply:
For all the others we striped the `protoc-` part, so let's stay with that and make it `envoy-gen-validate`?